backend/src/main/java/com/htecgroup/SkyNest/repository/RoleRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.htecgroup.SkyNest.repository;
+
+import com.htecgroup.SkyNest.model.enitity.RoleEntity;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
+public interface RoleRepository extends CrudRepository<RoleEntity, UUID> {
```

Comment:
why somewhere CrudRepository and somewhere JpaRepository ?

Reply:
JpaRepository extends CrudRepository, PagingAndSortingRepository, QueryByExampleExecutor, and Repository.

I thought that for roles, we don't need most of these. Should we have every repository extend JpaRepository?